internal/gencli/testdata/root_file.want

```diff
@@ -0,0 +1,59 @@
+// Code generated. DO NOT EDIT.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var Verbose, OutputJSON bool
+var ctx context.Context
```

Comment:
I think we should inline `context.Background()`. Passing `ctx` implies some context propagation.

Reply:
I'm cool with inlining it.

I just initialized it globally, at the root-command level so that it didn't have to be generated in every command. 

Wdyt? Inline or leave it?

Reply:
I think we should either inline it or actually init it globally like `var ctx = context.Background()`.